.github/workflows/check-backport.yml

```diff
@@ -0,0 +1,37 @@
+name: Stable Backport Check
+on:
+  issue_comment:
+  pull_request_target:
+    types:
+      - labeled
+
+jobs:
+  check-backport:
+    name: Check Backport
+    if: ( github.event.issue.pull_request != '' && contains(github.event.comment.body, '@jellyfin-bot check backport') && github.event.comment.author_association == 'MEMBER' ) || github.event.label.name == 'stable backport'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify as seen
+        uses: peter-evans/create-or-update-comment@v1.4.5
+        with:
+          token: ${{ secrets.JF_BOT_TOKEN }}
+          comment-id: ${{ github.event.comment.id }}
+          reactions: '+1'
+
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.JF_BOT_TOKEN }}
```

Comment:
you could use the 'default' `GITHUB_TOKEN` or none here IIRC

Reply:
I found something was needed and @ferferga mentioned the bot token, so I went with that.